iothub/service/src/DirectMethod/Models/DirectMethodServiceRequest.cs

```diff
@@ -58,51 +61,10 @@ public DirectMethodServiceRequest()
         [JsonIgnore]
         public TimeSpan? ResponseTimeout { get; set; }
 
-        /// <summary>
-        /// Get the payload object. May be null or empty.
-        /// </summary>
-        [JsonIgnore]
```

Comment:
The service client shouldn't need to fetch the payload back that it set. If it wants it, it can cast it back or serialize it to JSON (for logging).